processor/samplingprocessor/tailsamplingprocessor/processor.go

```diff
@@ -131,11 +131,39 @@ func getPolicyEvaluator(logger *zap.Logger, cfg *PolicyCfg) (sampling.PolicyEval
 	case RateLimiting:
 		rlfCfg := cfg.RateLimitingCfg
 		return sampling.NewRateLimiting(logger, rlfCfg.SpansPerSecond), nil
+	case Composite:
+		rlfCfg := cfg.CompositeCfg
+		return newCompositeFromCfg(logger, rlfCfg)
 	default:
 		return nil, fmt.Errorf("unknown sampling policy type %s", cfg.Type)
 	}
 }
 
+func newCompositeFromCfg(logger *zap.Logger, config CompositeCfg) (sampling.PolicyEvaluator, error) {
+	var SubPolicyEvalParams []sampling.SubPolicyEvalParams
+	rateAllocationsMap := getRateAllocationMap(config)
+	for i := range config.PolicyCfgs {
+		policyCfg := &config.PolicyCfgs[i]
+		policy, _ := getPolicyEvaluator(logger, policyCfg)
+
+		evalParams := sampling.SubPolicyEvalParams{
+			Evaluator:         policy,
+			MaxSpansPerSecond: rateAllocationsMap[policyCfg.Name],
```

Comment:
Isn't the value retrieved here the percents rather than actual number of spans (see line 162)?

Reply:
Its actually SPS not percent. Let me fix the name at line 162